lib/awful/titlebar.lua

```diff
@@ -108,6 +114,9 @@ local function new(c, args)
     -- Make sure the titlebar has the right colors applied
     bars[position].update_colors()
 
+    -- Handle declarative/recursive widget container
+    ret.setup = base.widget.setup
```

Comment:
lib/awful/titlebar.lua:117:17: accessing undefined variable base

(I think the default config should enable titlebars, not disable them; that way travis would have caught this...)